src/pages/Inventory.page.js

```diff
@@ -29,4 +31,67 @@ export class InventoryPage extends BaseSwagLabPage {
     async addItemToCartById(id) {
         await this.addItemToCartBtns.nth(id).click();
     }
+
+    async addItemToCartByIndex(index) {
+        const itemDiv = this.inventoryItems.nth(index);
+        const addButton = itemDiv.locator('.btn_inventory').first();
+        await addButton.click();
+    }
+
+    async getItemInfoById(id) {
+        const name = await this.inventoryItemName.nth(id).textContent();
+        const price = parseFloat((await this.inventoryItemPrice.nth(id).textContent()).replace('$', ''));
+        const description = await this.page.locator('.inventory_item_desc').nth(id).textContent();
+        return { name, price, description };
+    }
+
+    async getItemInfoByIndex(index) {
+        const name = await this.inventoryItemName.nth(index).textContent();
+        const price = parseFloat((await this.inventoryItemPrice.nth(index).textContent()).replace('$', ''));
+        const description = await this.page.locator('.inventory_item_desc').nth(index).textContent();
+        return { name, price, description };
```

Comment:
I did not get the purpose of this method. this code is the same as we have in the previous method